projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3913,6 +3913,11 @@ public function upload_handler( $update_media_item = false ) {
 			return new WP_Error( 'unknown_token', 'Unknown Jetpack token', 403 );
 		}
 
+		$upload_blocked = apply_filters( 'jetpack_upload_handler_block', false, $_FILES );
+		if ( is_wp_error( $upload_blocked ) ) {
```

Comment:
Nitpick (non-blocking): `jetpack_upload_handler_block` can set the wrong expectation that returning a `true` value  will also block the file upload, since the default value of the filter is `false`.

Do you think that using something more _WordPress-y_,  like `jetpack_upload_handler_can_upload` where `true` (default) proceeds with the upload and `false` or `WP_Error` blocks it, could help setting better expectations?